shared/src/balancer_event_handler.rs

```diff
@@ -78,8 +253,16 @@ impl BalancerPools {
                     ContractEvent::PoolRegistered(event) => {
                         Some(convert_pool_registered(&event, &meta))
                     }
+                    ContractEvent::TokensRegistered(event) => {
+                        Some(convert_tokens_registered(&event, &meta))
+                    }
+                    // ContractEvent::TokensDeregistered(event) => {
```

Comment:
remove?

Reply:
Sure, but I am going to handle this case immediately after in a follow up PR.